src/widget/form/settings/avform.cpp

```diff
@@ -271,7 +271,7 @@ void AVForm::fillCameraModesComboBox()
         qDebug("width: %d, height: %d, FPS: %f, pixel format: %s\n", mode.width, mode.height, mode.FPS, pixelFormat.toStdString().c_str());
 
         if (mode.height && mode.width)
-            str += QString("%1p").arg(mode.height);
+            str += QString("%1p").arg(getModeSize(mode));
```

Comment:
This is wrong, it causes the different resolutions to be shown as "4p" instead of "420p".